pages/common/jhipster.md

```diff
@@ -0,0 +1,36 @@
+# jhipster
+
+> Generates a monolithic or microservices project after prompting user.
+> More information: <https://www.jhipster.tech/>.
+
+- Generate a simple full-stack project (monolithic or microservices):
+
+`jhipster`
+
+- Generate a simple frontend project (with OPTION = --skip-server) or a simple backend project (with OPTION = --skip-client):
```

Comment:
These two OPTIONs can be split in two examples. Since then there would be too many examples in this page, maybe you want to move some of the subcommands into their respective pages? (e.g. `jhpister-upgrade` can have example from that subcommand)

Reply:
As `jhipster upgrade` [has no specific option](https://www.jhipster.tech/upgrading-an-application/), I'd rather prefer to let it that way if it's not a problem :)